python/cucim/src/cucim/skimage/feature/tests/test_blob.py

```diff
@@ -0,0 +1,560 @@
+import math
+
+import cupy as cp
+import pytest
```

Comment:
can just generate disks and ellipsoids on the CPU

```suggestion
import numpy as np
import pytest
from skimage.draw import disk, ellipsoid
```